tgui/packages/tgui/interfaces/Autolathe.js

```diff
@@ -80,20 +80,8 @@ export const Autolathe = (props, context) => {
                 </Button>
                 {recipe.is_stack === 1 && (
                   <Box as="span">
-                    <Button
-                      color={recipe.hidden && "red" || null}
-                      disabled={!canBeMade(recipe, materials, 5)}
-                      onClick={() => act("make", { make: recipe.ref,
-                        multiplier: 5 })}>
-                      x5
-                    </Button>
-                    <Button
-                      color={recipe.hidden && "red" || null}
-                      disabled={!canBeMade(recipe, materials, 10)}
-                      onClick={() => act("make", { make: recipe.ref,
-                        multiplier: 10 })}>
-                      x10
-                    </Button>
+                    {[5, 10, 25, 50]
+                      .map(mult => MultButton(recipe, materials, act, mult))}
```

Comment:
```suggestion
                    {[5, 10, 25, 50].map(mult => MultButton(recipe, materials, act, mult))}
```
А разве так не лучше?

Reply:
И я правильно понимаю, что это в итоге будет некий цикл, где каждая цифра вызовет MultButton и является mult ?

(я незнаю синтаксиса джиеса, так что ориентируюсь на опыт)

Reply:
Ок.